example/index.js

```diff
@@ -9,12 +9,13 @@ AppRegistry.registerComponent('example', () => App);
 
 TrackPlayer.registerEventHandler(async (data) => {
   if (data.type === 'playback-track-changed') {
-    const track = await TrackPlayer.getTrack(data.nextTrack);
-    TrackStore.title = track.title;
-    TrackStore.artist = track.artist;
-    TrackStore.artwork = track.artwork;
+    if (data.nextTrack) {
+      const track = await TrackPlayer.getTrack(data.nextTrack);
+      TrackStore.title = track.title;
+      TrackStore.artist = track.artist;
+      TrackStore.artwork = track.artwork;
+    }
   } else if (data.type === 'playback-state') {
-    PlayerStore.playbackState = data.state;
```

Comment:
Was this causing an issue on Android @Guichaguri ? this is what updates the state label in the playlist view.

Reply:
No, it doesn't make sense to set the playbackState here and then set it again right after it. (Lines 19 and 21)